realm/values/google_account_id.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is it OK to have this in our public repo? I'm guessing yes, but just confirming

Reply:
Yes, it's public information - but thanks for checking! (This kind of thing scares me silly)